pkg/server/handler.go

```diff
@@ -132,15 +132,6 @@ func WithWorkspaceProjection(apiHandler http.Handler, shardVirtualWorkspaceURL *
 			logger = logger.WithValues("from", path.Join(cluster.Name.Path(), req.URL.Path), "to", newPath)
 			logger.V(4).Info("rewriting path")
 			req.URL.Path = newPath
-			if shardVirtualWorkspaceURL != nil {
```

Comment:
Note, a redirect here is incorrect; the correct fix is to proxy requests from inside the apiserver to /services. That's a huge change and will be in a separate PR.